src/cli/commands/lint.ts

```diff
@@ -14,7 +14,7 @@ import { getDefaultRulesetFile } from '../../rulesets/loader';
 import { isOpenApiv2, isOpenApiv3 } from '../../rulesets/lookups';
 import { oas2Functions, rules as oas2Rules } from '../../rulesets/oas2';
```

Comment:
Hopefully we can clear out this old stuff too? I would expect that with loading rulesets, and rulesets being able to have functions, that we dont need all this oas2Functions, oas3Functions, oas2Rules, oas3Rules, etc? Especially seeing as oas2Functions and oas3Functions are identical. 

We should move our tests over to then new stuff to avoid divergence between what we're testing and what we're suggesting people use.